### DISCUSSION TITLE
Get the pseudo rapidity distribution of the quarks from Z boson decay without any selection.

### DISCUSSION THREAD

**Discussion by ram1123** [ram1123/2l2Q_limitSettingTool#26] [Comments_HIG_23_001] [answered]
L155 why the eta restriction on the Z decay products in the resolved case?

**Comment by jialin-guo1** [comment, marked as answer]
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/05626282-e967-4f56-a603-a6c9b359d7e7)

most of Z quark are in |2.4|<eta

**Reply by ram1123** [reply]
Good, this already explains it. Is it easy to get the RECO level distribution of pseudo rapidity with all cuts except cut on pseudo rapidity?

If it's difficult or too time-consuming, then leave it.
